src/main/java/com/softserve/actent/model/entity/Event.java

```diff
@@ -0,0 +1,124 @@
+package com.softserve.actent.model.entity;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import com.softserve.actent.constant.StringConstants;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.validator.constraints.Length;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "events")
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    @NotBlank(message = StringConstants.TITLE_SHOULD_NOT_BE_BLANK)
+    @Length(max = 100, message = "Too long")
+    @Column(nullable = false, length = 100)
+    private String title;
+
+    @Length(max = 500, message = "Too long")
+    @Column(length = 500)
```

Comment:
We decided not to restrict data length at the moment.

Reply:
This does not affect the basic logic

Reply:
It's redundant, we already argued and voted on that.

Reply:
other some entity has that

Reply:
Just those which were merged before our discussion on that. Not a valid argument to do it after it.

Reply:
I do not consider it like a problem we will come back to this later anyway!

Reply:
So now you can just remove redundant restriction, as we'll return to this later.

Reply:
If we will return and set them, so why I must remove?

Reply:
Now we can't know what restrictions we'll set, nor we don't know for sure that there will be need for such restrictions, it's just an assumption.
Also, as I already said, we voted for that, as you proposed to do.

Reply:
done

Reply:
I still see constraints, maybe you didn't push your commits.